packages/vue-components/src/Searchbar.vue

```diff
@@ -124,8 +124,9 @@ export default {
           src,
           title,
           headingKeywords,
+          frontmatter,
         } = entry;
-        const keywords = entry.keywords || '';
+        const keywords = frontmatter.keywords || '';
```

Comment:
Do we have a guess why this was `entry.keywords` initially?

Our old repo that contains much of this file's history is also available [here](https://github.com/MarkBind/vue-strap.git) so this could be a good exercise in `git blame`-ing your way through the diff.

Reply:
In my opinion, it was `entry.keywords` initially to prepare the `Searchbar` to include frontmatter keywords since there was no `keywords` property in the entry object. This was done to make it easier to implement the inclusion of frontmatter keywords in `Searchbar`. I came to this conclusion after looking through the history of commits and entry never contained a `keywords` property from the beginning of the commit history.

Reply:
> I came to this conclusion after looking through the history of commits and entry never contained a keywords property from the beginning of the commit history.

Agreed, this should be good enough. Thanks for investigating!